_includes/home-nav.html

```diff
@@ -1,8 +1,8 @@
 <div class="nav">
-    <ul>
-        <li class="home"><a href="#">Activities</a></li>
-        <li class="about"><a href="/projects">Projects</a></li>
-        <li class="news"><a href="#">Blog</a></li>
-        <li class="contact"><a href="#">Archives</a></li>
-    </ul>
+  <ul>
+    <li class="home"><a href="/category/activities">Activities</a></li>
```

Comment:
Add underline css effect on mouse over for categories

Reply:
okay, underline with red color on hover. is that okay ?

Reply:
okay, let's see how the red color looks :)